htdocs/api/v0.0.1/Login.php

```diff
@@ -129,6 +133,13 @@ function getEncodedToken($user)
                  );
 
         $key = $config->getSetting("JWTKey");
+        if ($key === "S3cret") {
```

Comment:
this should probably have `|| empty($key)` too.

Maybe add a `checkTokenStrength($key)` that returns a boolean and move the statement to there? That way additional key strength checks would have an obvious place to go in the future.